packages/pg/package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   name: 'simple:pg',
-  version: '0.0.1',
+  version: '0.0.2',
```

Comment:
No need to change the version since it's not published, but we can do it anyway :]